keyboards/planck/keymaps/ariccb/Layout.txt

```diff
@@ -0,0 +1,65 @@
+BASE LAYER
```

Comment:
Please delete this file (or place it in a readme).

Reply:
will do.